markdown-pages/en/tidb/_docHome.md

```diff
@@ -8,15 +8,23 @@ hide_leftNav: true
 <DocHomeContainer title="PingCAP Archived Documentation" subTitle="Access the archived documentation of the TiDB database in this website" archive>
 
 <DocHomeSection label="TiDB" anchor="tidb" id="tidb">
+  
+This website provides archived TiDB documentation for the following legacy TiDB versions:
 
-The following table lists the archived documentation of the TiDB database. PingCAP encourages you to use [the latest LTS version](https://docs.pingcap.com/tidb/stable/) of the TiDB database. Archived documentation no longer receives updates.
+- TiDB Long Term Support (LTS) versions that are after End of Life (EOL) dates
```

Comment:
To keep it consistent with that in https://docs.pingcap.com/tidb/stable/versioning
```suggestion
- TiDB Long-Term Support (LTS) versions that are after End of Life (EOL) dates
```